### DISCUSSION TITLE
Eternity - AI Kpop女團！

### DISCUSSION THREAD

**Discussion by BlahFish1006** [1101-GS4719-C/Python-Community-GS4719-C#1112] [General]
Eternity 在2021年以[《我是真的（I'm Real）》單曲](https://youtu.be/XQMg8-Ku3JI) 出道，跟一般女團不一樣的是，這11位成員皆為虛擬人物！
Pulse9 這家科技企業最初生成了101張由深度學習產出的臉孔，透過粉絲票選出最愛的臉孔，再由設計師將她們動畫化。
她們可以唱可以跳，甚至也可以實現在直播聊天、視頻攝製和線上粉絲見面會。

我覺得她們公司最厲害的是，僅僅透過一年的時間就改善了非常多！
可以看她們在去年(2022) 10月新推出的[MV 《DTDTGMGN》](https://youtu.be/EYIdOXjwOOE)，可以明顯感受到相較於出道曲，
虛擬成員們的動作非常順暢、臉和眼神也都自然許多！不禁令人體會到科技進步更新的速度有多驚人。

虛擬藝人不但不會受制於身體體力限制，也不會容易因緊張、難過等情緒問題，而受到精神上的困擾。
就如我們所知，韓國的偶像藝人通常都會當過一段時間的練習生、受極嚴格的訓練後，才「有機會」能夠出道，
若未來有更多AI女團出道，是否會影響到更多練習生出道的機會呢？不禁覺得很心疼那些努力很久的孩子們。

文章連結：https://www.bbc.com/zhongwen/trad/world-63940314

**Comment by wenchi9023** [comment]
看了他們的mv，真的好像真人!
但如果要選擇，比起AI女團，我還是喜歡真人><
畢竟還是喜歡感受人與人之間的溫度，儘管AI能做到和真人幾乎一樣的事情，但感覺還是有差異!

**Comment by dinosaur1009** [comment]
AI的偶像要能讓大家都接受 好像還是有一點難度！但真的很酷！

**Comment by wanderer657** [comment]
在我看來，虛擬藝人的出現確實是一種創新，對我來說，它讓我重新思考了娛樂產業的未來發展方向。作為一個喜歡音樂和娛樂的人，我對這種新興形式感到興奮和好奇。我曾經在社交媒體上關注過一些虛擬藝人，並且對她們的表現和創作感到驚訝。透過這些虛擬藝人，我看到了科技在娛樂領域中的無限潛力，也期待著未來更多的創新和驚喜。

**Comment by SusuTima** [comment]
目前對於AI偶像的接受度還是很低，雖然他們看起來很真實，但還是更喜歡看到真人在舞台上表演和他們日常生活中的一些花絮。

**Comment by Eileen050824** [comment]
因為身邊有朋友在追PLAVE，後來才知道這個是個虛擬男團，我原以為AI偶像和其作品會很尷尬，但卻出乎意料地自然，完全展示了開發者的辛勤付出。但我還是認為虛擬偶像無法完全取代真人，畢竟她們的行動都在程式計算的預料之中，缺乏真正K-POP的人格魅力。也認為這個行業是由一群有夢的男孩女孩辛苦踩出的道路，情感上更為他們感到不捨！

**Reply by woohsin** [reply]
PLAVE的歌都很好聽！！
他們出道曲我刷了好多次，爾且MV做的超精緻，不知道燒了多少錢，而且跟V一樣都會超常開播，我是把他們當作高經費V團啦哈~

**Comment by wanka2030** [comment]
看到AI偶像後，就想起來那時候再看lol世界賽的時候，由KDA所呈現的偶像女團給人一種煥然一新的感覺，但有可能因為當時的全息投影不太熟練，後面還是需要真人歌手的演唱來撐場，但如果以現在的技術來說的話，一定能夠給出更加厲害也更加華麗的舞台show❤️

**Comment by jeff110303549** [comment]
我覺得虛擬偶像作為科技發展的一個產品相當厲害，充分考慮到了當下大眾流行音樂裡的商機，其成果也是超乎我想像的優秀，不過我想應該有很多人追Kpop團不只是為了他們所提供的音樂，更是從他們身上看到了即便是偶像也能很有溫度，因而產生了更多的親近感，這種難以用程式再現的人情或許會是虛擬偶像面臨的一個大困難吧

**Comment by cassy0201** [comment]
我覺得真實的偶像團體還是有存在的必要，畢竟他們能在現實生活中與粉絲見面、交流互動，同時也為粉絲帶來情緒價值，這是目前AI偶像比較難達成的

**Comment by 710388** [comment]
沒聽過過，但感覺是個蠻新穎的東西，也許可以讓每個人都跟自己的偶像見面？

**Comment by yubo0oo** [comment]
最頂的還得是金采原

**Comment by MingWei174** [comment]
比起AI女團，我更喜歡現實在演唱會可以跟你互動的真實idol，對於去每個國家巡迴都有不同的感想可以跟粉絲們分享，那種真情留露的感覺是AI無法創造出來的，btw小弟我想分享我自己最喜歡的幾個K-pop女團及本命歌手
1.IVE(張員瑛  安俞真  Liz)
2.NewJeans(Hanni  Minji  Haerin)
3.le sserafim(洪恩採)
4.(G)-idle(Minnie  麵姐)
5.ITZY(Lia  有那)
6.BabyMonster(Ahyeon)
7.IU
8.aespa(Winter  Karina)
9.Gyubinnnnn
10.Taeyeon

**Comment by hunteryang0108** [comment]
偶像是一種很複雜的文化，我認為AI很難取代真人偶像，不過還是謝謝同學的分享。人們追隨特定偶像往往是因為人物的個性或是一些個性特色，這是人工製造的AI偶像無法取代的地方。

**Comment by Nana1218** [comment]
虛擬偶像是一個很創新的想法，但以我的個人經驗來說，AI終究是無法超越真人偶像的，也許AI偶像不會累，不需要經過長時間的練習生時期，但我之所以會喜歡真人偶像，就是被他們的精神激勵了，在練習生時期所經歷的一切，抑或是從剛出道慢慢累積人氣的過程，又或是慢慢學習如何學習成為更好的偶像，這些過程是AI偶像不會有的，AI偶像在很多方面都很完美，但追隨偶像時還是會希望能看到他們人性化的一面。
